Cesium.Runtime/StringFunctions.cs

```diff
@@ -14,26 +16,8 @@ public unsafe static class StringFunctions
 {
     public static nuint StrLen(byte* str)
     {
-#if NETSTANDARD
-        if (str == null)
-        {
-            return 0;
-        }
-
-        Encoding encoding = Encoding.UTF8;
-        int byteLength = 0;
-        byte* search = str;
-        while (*search != '\0')
-        {
-            byteLength++;
-            search++;
-        }
-
-        int stringLength = encoding.GetCharCount(str, byteLength);
```

Comment:
This could produce incorrect behavior - `strlen` is index of \0, GetCharCount calculates the length in UTF-16 code units, which will differ should the bytestring contain non-ASCII chars.